AnimatedTextInput/Classes/AnimatedTextInputStyle.swift

```diff
@@ -25,8 +26,9 @@ public struct AnimatedTextInputStyleBlue: AnimatedTextInputStyle {
     public let activeColor = UIColor(red: 51.0/255.0, green: 175.0/255.0, blue: 236.0/255.0, alpha: 1.0)
     public let placeholderInactiveColor = UIColor.gray.withAlphaComponent(0.5)
     public let inactiveColor = UIColor.gray.withAlphaComponent(0.5)
-    public let lineInactiveColor = UIColor.gray.withAlphaComponent(0.2)
     public let lineActiveColor = UIColor.gray.withAlphaComponent(0.0)
+    public let lineInactiveColor = UIColor.gray.withAlphaComponent(0.2)
+    public let lineHeight: CGFloat = 1.0
```

Comment:
Please use: 
`public let lineHeight: CGFloat = 1.0 / UIScreen.main.scale` to maintain original value.